app/controllers/dashboard/cards_controller.rb

```diff
@@ -1,3 +1,5 @@
+require 'super_memo_service'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.